### ISSUE TITLE
Base model train param

### ISSUE THREAD

**Issue by SipengWang** [MCZhi/DTPP#10]
Thank you for your excellent work！I tried your published base model and it performed well. Since I want to reproduce this base model, what train param do i need to follow? Param in paper or in code or others?

**Comment by MCZhi** [comment]
Hi, @SipengWang, thank you for your interest. Yes, you can use the default params provided by the code or you can increase the amount of training data.

**Comment by SipengWang** [comment]
> Hi, @SipengWang, thank you for your interest. Yes, you can use the default params provided by the code or you can increase the amount of training data.

Thank you for your replay, so the training data for base model is 100k scenarios sampled from valid dataset? as you mentioned in your paper

**Comment by MCZhi** [comment]
Yes, around 100k scenarios from the validation set.
